src/main/webapp/assets/css/feedStyle.css

```diff
@@ -376,6 +376,90 @@ select {
   opacity: 0.4;
 }
 
+/*
+ * Filters (button + card)
+ */
+
+#toggle-filters-button {
+  background-color: var(--dark-maroon);
+  border: 1px solid transparent;
+  border-radius: 50px;
+  box-shadow: 0 0 10px lightgrey;
+  color: white;
+  font-size: 1.25rem;
+  font-weight: 400;
+  height: 60px;
+  margin: 0 0 0 1rem;
+  outline: none;
+  padding: 0 30px;
+}
+
+#toggle-filters-button:hover {
+  background-color: white;
+  border: 1px solid var(--dark-maroon);
+  color: var(--dark-maroon);
+  cursor: pointer;
+}
+
+.keyboard-active #toggle-filters-button:focus {
+  background-color: #ff9595;
+}
+
+#filter-card {
+  border: 1.5px #adadad solid;
+  border-radius: 15px;
+  display: none;
+  margin-bottom: 5px;
+  margin-top: 20px;
+  min-height: 100px;
+  min-width: 100px;
+  padding: 10px;
+  padding-right: 30px;
+  padding-left: 30px;
```

Comment:
padding: 10px 30px 10px 30px?

Reply:
Yeah I just stole the formatting from the `post-card` element so they would match in shape/behavior :)  I can fix it up though!

Reply:
Also I believe `padding: 10px 30px` does the same thing :)